src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router-dom';
+import { IonApp, IonRouterOutlet } from '@ionic/react';
+import { IonReactRouter } from '@ionic/react-router';
+import Home from './pages/Home';
+import '@ionic/react/css/core.css';
+import '@ionic/react/css/normalize.css';
```

Comment:
are these necessary?

Reply:
- Normalize.css makes browsers render all elements more consistently and in line with modern standards. It precisely targets only the styles that need normalizing.
-  core .css import of the core CSS for Ionic .
- Depending on the furthur necessaries we can import or deleate the necessary components.